report_txt/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Qweb Text Reports
+=================
+
+This module extends the possibilities of the native Qweb reporting engine to be able to generate any kind of text files (TXT, CSV, ZPL, EPL, ...).
+
+Usage
+=====
+
+To add a text report, you should develop an Odoo module that contains:
+
+* a Qweb template,
+* a Qweb report definition with *report_type = qweb-txt*.
+
+The module *purchase_dilicom_csv* available on the `dilicom Github repository <https://github.com/akretion/dilicom>`_ (branch *10.0*) is a good example: it adds a report *Dilicom CSV Order* on purchase orders. This report is a CSV file with one line per order line and 2 columns: EAN13 and order quantity.
```

Comment:
Instead of this, you should generate a demo report with base objects. For example, a res.users report.